ProcessQueue/__init__.py

```diff
@@ -30,76 +31,119 @@ def process_scan_report_sheet_table(sheet):
     """
     results = []
     # Get max number of columns in the sheet
-    max_column=sheet.max_column
+    max_column = sheet.max_column
     # Set a column index (openpyxl index at 1 instead of 0)
-    col_idx=1
-    
-    for row_idx,row_cell in enumerate(sheet.iter_rows(min_row=1,max_col=max_column),start = 1):
-  
-    # Works through pairs of value/frequency columns
-        for column_idx,cell in enumerate(row_cell):
+
+    for row_idx, row_cell in enumerate(
+        sheet.iter_rows(min_row=1, max_col=max_column), start=1
+    ):
+
+        # Works through pairs of value/frequency columns
+        for column_idx, cell in enumerate(row_cell):
+            freq = sheet.cell(row=row_idx, column=column_idx + 2).value
             if cell.value:
                 if (column_idx) % 2 == 0:
-                # As we move down rows, checks that there's data there
-                # This is required b/c value/frequency col pairs differ
-                # in the number of rows
-                  if sheet.cell(row=row_idx,column=column_idx+1).value == "" and sheet.cell(row=row_idx,column=column_idx+2).value == "":
-                    continue
-                # Append to Results as (Field Name,Value,Frequency)
-                  results.append(
-                      (sheet.cell(row=1,column=column_idx+1).value,sheet.cell(row=row_idx,column=column_idx+1).value, sheet.cell(row=row_idx,column=column_idx+2).value)
-                  )
-    col_idx=col_idx+1
+                    # As we move down rows, checks that there's data there

```

Comment:
Please move `if (column_idx) % 2 == 0:` out to before `if cell.value:`: in both branches  of the `if cell.value / else` we only do something if `(column_idx) % 2 == 0`, so it's clearer to check for it in only one place.

Reply:
In fact, I think this whole thing can still be simplified.
```
if sheet.cell(row=row_idx, column=column_idx + 1).value or sheet.cell(row=row_idx, column=column_idx + 2).value:
  results.append(
                        (
                            sheet.cell(row=1, column=column_idx + 1).value,
                            sheet.cell(row=row_idx, column=column_idx + 1).value,
                            sheet.cell(row=row_idx, column=column_idx + 2).value,
                        )
                    )
```
or am I missing something? This can be made even clearer if we set `start=1` in the `enumerate(row_cell)` to 
```
if sheet.cell(row=row_idx, column=column_idx).value or sheet.cell(row=row_idx, column=column_idx + 1).value:
  results.append(
                        (
                            sheet.cell(row=1, column=column_idx).value,
                            sheet.cell(row=row_idx, column=column_idx).value,
                            sheet.cell(row=row_idx, column=column_idx + 1).value,
                        )
                    )
```

Reply:
I think I tried this before:
if sheet.cell(row=row_idx, column=column_idx).value or sheet.cell(row=row_idx, column=column_idx + 1).value:
 and it was giving me bad request at the values, but that was back when we weren't allowing blank value names I think so that might be why. I can try to change it around see if we get the same results